src/js/main.js

```diff
@@ -8,6 +8,70 @@ TODO: add UI as a dependency once installed
 
 //IFFE
 ;(function () {
-  angular.module('cropswap', ['ngRoute'])
+  var BASE_URL = "http://cropswap.herokuapp.com/";
+
+  angular.module('cropswap', ['ngRoute'], function($routeProvider) {
+    $routeProvider
+      .when('/', {
+        redirectTo: '/signup'
+      })
+      .when('/signup', {
+        templateUrl: 'partials/signup.html',
+        controller: function($http, $location) {
+          var signup = this;
+
+          signup.user = {
+
+          };
+
+          signup.signupUser = function () {
+            console.log('it works!');
+            $http.post(BASE_URL + "users.json" + "email", signup.user)
+              .then(function() {
+                console.log('it works');
+                $location.path('/newact')
+              });
+          };
+        },
+        controllerAs: 'signup'
+      })
+      .when('/newact', {
+        templateUrl: 'partials/newact.html',
+        controller: function($http, $location) {
+          var newact = this;
+
+          newact.user = {
+
+          };
+
+        newact.newactuser = function () {
+          console.log('its working');
+          $http.patch(BASE_URL + "users/user:id.json" + "name", newact.user)
```

Comment:
Talking to David: 
His tips: 
1) Use string concatenation for the base part of the URL
2) If you need a dynamic part of the URL, like User ID, it has to be a variable. Something like user:id which we're using now will never change. He thinks there is already a user:id variable we can use and just concantonate in. 
3) Parameters should be sent via json. I tested this in Postman and it works, so if you can send params via json I can take them. 

He said you guys did some examples in class on get and post, for Github and Etsy.